index.html

```diff
@@ -239,7 +239,7 @@ <h2>Scope</h2>
           <li>
 Concentrate their efforts on the identified
 <a href="https://w3c-ccg.github.io/did-use-cases/">use cases</a>
```

Comment:
Looks like accidental delete?

Reply:
Why do you think that text is deleted? Because it's in red? That's funky Github markup.